addons/sale/wizard/sale_make_invoice_advance.py

```diff
@@ -220,6 +219,11 @@ def _prepare_down_payment_section_values(self, order):
 
     def _prepare_so_line_values(self, order):
         self.ensure_one()
+        analytic_distribution = {}
+        for line in order.order_line:
+            distrib_dict = line.analytic_distribution or {}
+            for account, distribution in distrib_dict.items():
+                analytic_distribution[account] = distribution + analytic_distribution.get(account, 0)
```

Comment:
Shouldn't this be in weighted compared to the total amount of the lines?

Reply:
I would expect so, but it is not what the sales team have done (they were just putting the all the tags on the so_line). They even revamped it recently and kept this behaviour.

Reply:
Well, we revamped the wizard to clean its code and logic, but we don't make any change to `account` stuff tbh.
If William says it should be weighted, probably makes sense (and I doubt there was any way to do that previously with the tags tbh).

Reply:
Done then